app/tests/spec/lib/fxa-client.js

```diff
@@ -821,7 +779,7 @@ function (chai, $, sinon, FxaClient, p, testHelpers, FxaClientWrapper, AuthError
         sinon.stub(realClient, 'accountUnlockResendCode', function () {
           return p();
         });
-        var triesLeft = Constants.ACCOUNT_UNLOCK_RESEND_MAX_TRIES;
+        var triesLeft = Constants.EMAIL_RESEND_MAX_TRIES - 1;
 
         // exhaust all tries
         var promises = [];
```

Comment:
This test should no longer be necessary, and [this assertion](https://github.com/mozilla/fxa-content-server/pull/2553/files#diff-c4fce5a20a1f1b633ee88c427c1a7acdR800) should definitely not pass, `Constants.ACCOUNT_UNLOCK_RESEND_MAX_TRIES` no longer exists.